### ISSUE TITLE
[Suggestion] Edits to the Intro to Accessibility page

### ISSUE THREAD

**Issue by jeremyzilar** [GSA/digitalgov.gov#1625]
The Section 508 team in OGP passed over a few edit suggestions for our **Intro to Accessibility** page — https://digital.gov/resources/intro-accessibility/

I am logging them here so we have a consistent place to see them all and decide which to fix 👀 

---
**From Michael Horton:**

> I would like to ask that the URL used in the first sentence of the Accessibility is the Law section, "Section 508 Standards are established and maintained..." be changed to https://www.access-board.gov/guidelines-and-standards/communications-and-it/about-the-ict-refresh. The reason is that the page the article currently directs users to and that has a big "508 standards" link on it directs user to the old standards. I've contacted that Access Board and they've indicated that they are aware of the issue and working on addressing the issue.   

> Additionally, I'd be remiss to not share a few accessibility issues on the page which may impact a user's ability to access all of the information thats been made available in the article:

> - the captions under the MS design toolkit images are below contrast standard minimums
> - the social media links in the upper right-hand corner lack descriptive text
> - the Microsoft PDF file lacks full and proper tagging
> - the two UK Gov PDF posters lack full and proper tagging
> - the two embedded videos both lack Audio Description WCAG 1.2.3 which convey visual information 
> - the sites hyperlinks use only color to indicate that they are a hyperlink

> As always, I'm happy to help the team better understand these issues and discuss possible remediation steps.  

---
**From John Sullivan:**

> 1. Please list Section508.gov ahead of the two WCAG references
> 2. In addition please add the following under the Section.508.gov reference (as we already have basically three "Accessibility 101" training courses out there:

> "For more basics on Section 508, please also look at these online training opportunities:
> - Accessibility of ICT: An Overview for Government Executives – High-level overview of the Revised 508 Standards. Explains the roles and responsibilities of Federal government executives, including agency heads, CIOs and CAOs, and Section 508 Program Managers.
> - Section 508: What It Is and Why It's Important? – Introduction to Section 508 and Information and Communication Technology (ICT). Explains what Section 508 is and why it’s important, shows how conformance can make ICT more accessible, reviews job-related responsibilities for meeting Section 508 standards, offers resources to help you meet your Section 508 responsibilities.
> - Section 508 Awareness - What is a Section 508 Program? This course provides an overview of federal agency 508 programs, and how they help agencies deliver accessible technology products and services to employees and the public.  

---

**From Rachel Flagg:**

> To follow up on John's suggestions, here are some recommended bullets for those resources:

> - ICT Testing Baseline - Baseline tests emphasize methods and techniques to reliably meet Section 508 requirements
> - DHS Trusted Tester Process - A manual test approach that aligns with the ICT Testing Baseline, and provides repeatable and reliable conformance test results
> - ANDI (Accessible Name & Description Inspector) - Developed by SSA, ANDI is a free tool to test web content for accessibility

> Could you please add to the Recommended Resources section at the bottom of the page?

**Comment by jeremyzilar** [comment]
### For suggestions that involve "Accessibility Training"
We are putting together a page that lists out accessibility training opportunities that are available to be as a government employee. See https://github.com/GSA/digitalgov.gov/issues/1626

It is important that we not load this page with ALL the things. In fact, it probably needs to be about half the size it is now.

The focus of the page is just an introduction to accessibility. While it should have the right jumping off points for people to learn more, that is not the primary function of the page.

**Comment by jeremyzilar** [comment]
## From Michael's suggestions about issues on the site _(not in the page)_

> - the social media links in the upper right-hand corner lack descriptive text
> - the Microsoft PDF file lacks full and proper tagging
> - the sites hyperlinks use only color to indicate that they are a hyperlink

These will be fixed in the next release of the site.
